File: regex/T2.php
#!/usr/bin/php5 
#
<?php

if ($file = fopen("T2.txt", "r") ) {

    while(!feof($file)) {
        $string = fgets($file);
        preg_match_all('@[ctag]{6,}@i',
        "$string",
        $matches);
        print_r($matches);
    }
    fclose($file);
}

File: regex/T2.py
#!/usr/bin/python
# -*- coding: utf-8 -*-
import re

file = open('T2.txt', 'r')



for Fstring in file.readlines():
    p = re.compile(r'[ctag]{6,}',re.IGNORECASE)
    res = p.findall(Fstring,0)
    print res
    res=""
